src/Directory.Build.props

```
@@ -25,6 +25,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="PolySharp" Version="1.10.0" Condition="'$(TargetFramework)' != 'net5' AND '$(TargetFramework)' != 'net6' AND '$(TargetFramework)' != 'net7'">
```

Isn't the point here that PolySharp determines what polyfills are necessary based on which .NET version is being referenced?

```suggestion
    <PackageReference Include="PolySharp" Version="1.10.0">
```

---

It does indeed. This condition should not be necessary.

---

You may try. I tried it at first ang got compilation errors.

---

OK, I got the reason.